packages/react-sdk/src/expression.test.ts

```diff
@@ -0,0 +1,98 @@
+import { expect, test } from "@jest/globals";
+import { executeExpression, validateExpression } from "./expression";
+
+test("allow literals and array expressions", () => {
```

Comment:
can we put that into describe('expression')?

Reply:
Jest already groups by module name. This will just add additional nesting without any other benefit.